Automattic/wp-calypso
# Show site preview in Migration instructions page using screenshots

Related to https://github.com/Automattic/wp-calypso/issues/91964

## Proposed Changes

* Using the mShot service to generate screenshots of source sites for previewing on the Migration Instructions page

## Why are these changes being made?


* All sites don't allow loading them in iFrames

## Testing Instructions



* Activate the feature flag: `migration-flow/new-migration-instructions-step`.
* Navigate through the flow to import a site, and in the final step make sure you see the new migration instructions screen.
* Check that you can see a preview of the site
* Make sure the preview is an image
* Now resize the screen to be Tab sized
* Check that you see a new image generated that's Tab sized
* Check the same for Mobile.

https://github.com/Automattic/wp-calypso/assets/6820724/43794041-5390-47c2-9bae-507a3590e72f


## Pre-merge Checklist
